docker-entrypoint.sh

```diff
@@ -142,27 +145,15 @@ set_apache_server_name
 wait_for_db
 
 # Run database migrations
-if [ $(mysql -u "$DB_USERNAME" -p"$DB_PASSWORD" -h "$DB_HOST" -D "$DB_DATABASE" -N -s -e "SELECT COUNT(*) FROM information_schema.TABLES WHERE TABLE_SCHEMA = '$DB_DATABASE' AND TABLE_NAME = 'plants';") -eq 1 ]
-then
-    echo "Running unapplied database migrations..."
-    php asatru migrate:list
-else
-    echo "Running full database migrations..."
-    php asatru migrate:fresh
-fi
+echo "Running database migrations..."
+php asatru migrate:fresh
```

Comment:
Hi! I'm not sure if it's a good idea to run `migrate:fresh` on every start of the container as this seems to drop and recreate all tables? Doesn't this lead to dataloss?
See `migrate_fresh()` [here](https://github.com/danielbrendel/dnyAsatruPHP-Framework/blob/410b2fe4710bce9c4e5f662e640f12b864e5cbfb/src/database.php#L1205), `dropAll()` [here](https://github.com/danielbrendel/dnyAsatruPHP-Framework/blob/410b2fe4710bce9c4e5f662e640f12b864e5cbfb/src/database.php#L1139) and the `down()` functions of the migrations in `/app/migrations`.

Reply:
I agree! Ironically enough i was the one to propose the change initially.  I've reverted the deletion.